tutorial/README.md

```diff
@@ -0,0 +1,413 @@
+             .write.format("com.databricks.spark.redshift")
+            .option("url", jdbcURL)
+            .option("tempdir", tempS3Dir)
+            .option("dbtable", "redshiftevent")
+            .mode(SaveMode.Overwrite) 
+            .save()
+/*
+ *Append to an existing table redshiftevent if it exists or create a new one if it does not
+ *exist and write event records with event id greater than 1000
+ */
+
+sqlContext.sql("select * from myevent where eventid>1000").withColumnRenamed("eventid", "id")
+          .write.format("com.databricks.spark.redshift")
+          .option("url", jdbcURL) 
+          .option("tempdir", tempS3Dir) 
+          .option("dbtable", "redshiftevent") 
+          .mode(SaveMode.Append)
+          .save()
+```
+Listing 5: Writing to Redshift tables
+
+spark-redshift automatically creates a Redshift table with the appropriate schema determined from the table/DataFrame being written. The default behavior is to create a new table and to throw an error message if a table with the same name already exists (corresponding to `SaveMode.ErrorIfExists`)
+
+There are two key points to note in Listing 5-
+
+1. Note the `.withColumnRenamed("eventid", "id")`. This feature if necessary if any of the source table columns names are key words in Redshift (ex. table) then spark-redshift will throw and error. This will be [fixed](https://github.com/databricks/spark-redshift/issues/80) in the next release 
+2. Note how we utilize the modes. The first write utilizes the mode `SaveMode.Overwrite` which means that the table will be dropped and recreated it exists. The second query utilizes `SaveMode.Append` which will create the table if it does not exist but will append to it if it does. The default mode is `SaveMode.ErrorIfExists` which creates the table if it does not exists and throws and error if it does. The last mode is `SaveMode.Ignore` which is same as `SaveMode.Overwrite` if the table does not exist but does nothing if the table exists.
+
+While the above examples used Scala we could have also used SQL. And example of writing to a Redshift table in SQL is shows in Listing 6. The only thing to be aware of in the SQL mode is that SaveMode's are not available and only the default mode (`SaveMode.ErrorIfExists`) is in force.
+
+```sql
+> %sql
+CREATE TABLE redshiftevent
+USING com.databricks.spark.redshift
+OPTIONS (
+dbtable 'redshiftevent',
+tempdir 's3n://redshift-spark/temp/',
+  url 'jdbc:redshift://swredshift.czac2vcs84ci.us-east-1.redshift.amazonaws.com:5439/sparkredshift?user=spark&password=mysecretpass'
+)
+AS SELECT * FROM myevent;
+```
+
+By default the save operation uses the key distribution [style](http://docs.aws.amazon.com/redshift/latest/dg/c_choosing_dist_sort.html) of `EVEN` in Redshift. This can be changed by the use of option parameter `diststyle` and `distkey`. See spark-redshift [documentation](https://github.com/databricks/spark-redshift) for details.
```

Comment:
"Strunk & White fan" - Didn't know what that was. Spent an inordinate time online trying to make sense of it :-). Fixed it as you suggested :-).